README-FIPS.md

```diff
@@ -1,20 +1,39 @@
 OpenSSL FIPS support
 ====================
 
-This release of OpenSSL includes a cryptographic module that is intended to be
+This release of OpenSSL includes a cryptographic module that can be
 FIPS 140-2 validated. The module is implemented as an OpenSSL provider.
 A provider is essentially a dynamically loadable module which implements
 cryptographic algorithms, see the [README-PROVIDERS](README-PROVIDERS.md) file
 for further details.
 
-The OpenSSL FIPS provider comes as shared library called `fips.so` (on Unix)
+A cryptographic module is only FIPS validated after it has gone through the complex
+FIPS 140 validation process. As this process takes a very long time, it is not
+possible to validate every minor release of OpenSSL.
+If you need a FIPS validated module then you must ONLY generate a FIPS provider
+using OpenSSL versions that have valid FIPS certificates. A FIPS certificate
+contains a link to a Security Policy, and you MUST follow the instructions
+in the Security Policy in order to be FIPS compliant.
+See <https://www.openssl.org/source/> for information related to OpenSSL
+FIPS certificates and Security Policies.
```

Comment:
I'm not seeing anything on that page mentioning FIPS certificates or Security Policies. Will that page be updated, or will this link be changed?

Reply:
In all seriousness, where is the list of approved versions?

Reply:
Currently it is nowhere other than in a blog,,, That is what I am trying to fix..
There are PR's in the web repo also.

Reply:
https://github.com/openssl/web/pull/376

Reply:
This PR should be merged after the web related one.

Reply:
I would like the information to be accurate and up to date, which is why I just want to link to the other location.